includes/monk-functions.php

```diff
@@ -80,3 +80,114 @@ function monk_get_url_args( $arg ) {
 		return;
 	}
 }
+
+/**
+ * This function makes the array of available languages
+ *
+ * @since  0.4.0]
+ *
+ * @return array The available languages array.
+ */
+function get_monk_languages() {
```

Comment:
Wrap this function like the following example:
https://codex.wordpress.org/Transients_API#Complete_Example